docs/running-on-kubernetes.md

```diff
@@ -83,7 +83,7 @@ the cluster.
 ### Specifying input files
 
 Spark supports specifying JAR paths that are either on the submitting host's disk, or are located on the disk of the
-driver and executors. Refer to the [application submission](submitting-applications.html#advanced-dependency-management)
+driver and executors. Refer to the [application submission](submitting-applications.md#advanced-dependency-management)
```

Comment:
Thanks @johscheuer. I think we should make it evident that the namespace should exist, so, that change is valuable. 
If you revert the md->html changes, I'll gladly merge this.